### ISSUE TITLE
Tuner (and lowrider) Performance

### ISSUE THREAD

**Issue by Nick-Vyvyan** [fkk-cz/noire_vehicles#880]
These categories across the board under perform relative to their price, counterparts, and to other similar vehicles in the city. this is particularly bad for the tuners that are supposed to be _tuned_ versions of their real life counterparts. For example, some tuners have 8-speed transmissions but can never reach speeds to get above 6th gear, so whats the point? especially considering the recent recalls, I think these cars could use some love to allow more variety. 

in short, tuner performance should be greatly increased and (maybe?) reduce prices a bit. but for sure need performance boosts

thanks
